crates/key-server/key-server-config.yaml

```diff
@@ -14,6 +16,16 @@ set this to false for mvr to resolve on testnet.
 # 1. Set the `node_url` in this config file
 # 2. OR omit `node_url` here and set the NODE_URL environment variable
 #
+# The `seal_package` parameter for Custom and Devnet should point to what Seal package
+# deployment to use. This must be one of:
+# seal_package: Mainnet
+# seal_package: Testnet
+# seal_package: !Custom
+#   'Package id to use'
+# For Devnet, the parameter is mandatory.
+# For Custom, the parameter is optional, and if it is not set, the default Seal package
+# on Mainnet will be used by default. It can either be set in this config file or as an
+# environment variable, SEAL_PACKAGE.
```

Comment:
do we need to change this at runtime?

Reply:
hmm, why do we need the env var? let's make it mandatory in the config for Custom